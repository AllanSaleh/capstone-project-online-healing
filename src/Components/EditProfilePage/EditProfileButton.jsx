import React from 'react';

const EditProfileButton = ({ text, bgColor, color, margin }) => (
  <button
    type="button"
    className={`${bgColor || 'bg-blue-dark'}
     text-xl text-black lg:${margin} my-2 rounded-lg uppercase px-12 py-3 lg:text-subtitle border transition duration-300 hover:bg-white hover:${`border-${color}`}
    hover:${`text-${color}`}`}
  >
    {text}
  </button>
);
export default EditProfileButton;
