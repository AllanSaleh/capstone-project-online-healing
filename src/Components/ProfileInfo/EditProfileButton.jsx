import React from 'react';

const EditProfileButton = ({ text, bgColor, textColor, margin, bgOpacity }) => (
  <button
    className={`${bgColor} ${bgOpacity} ${textColor} ${margin} rounded-lg uppercase w-1/3 py-3 text-subtitle transition duration-300 hover:bg-opacity-100 hover:shadow-lg`}
  >
    {text}
  </button>
);

export default EditProfileButton;
