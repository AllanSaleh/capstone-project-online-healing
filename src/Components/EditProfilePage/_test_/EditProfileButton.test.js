import * as React from 'react';
import * as renderer from 'react-test-renderer';
import EditProfileButton from '../EditProfileButton';

it('Edit Profile Button renders correctly', () => {
  const tree = renderer.create(<EditProfileButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
