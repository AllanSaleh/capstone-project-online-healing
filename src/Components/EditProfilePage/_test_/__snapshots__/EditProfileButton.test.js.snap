// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="bg-blue-dark undefined text-black undefined rounded-lg uppercase px-12 py-3 text-subtitle transition duration-300 hover:bg-opacity-100 hover:shadow-lg"
  type="button"
/>
`;
